.github/workflows/image-canary.yaml

```diff
@@ -0,0 +1,25 @@
+name: ImageCanary
+on:
+  workflow_dispatch: {}
+  schedule:
+  - cron: '0 */4 * * *'
+jobs:
+  ImageCanary:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+      with:
+        ref: ${{ inputs.git_ref }}
```

Comment:
Shouldn't we always checkout from head here?

Reply:
So the git_ref would always be head, but I didn't realize that this input didn't exist when I copied it over. Surprised it ran in my fork, going to guess since it didn't resolve to anything it just used the default (i.e. main). Should be good now though, no need to specify a ref at all.